syntax = "proto3";

message Person {
  string id = 1;
  string first_name = 2;
  string last_name = 3;
  string company_name = 4;
}

message PersonList {
  repeated Person persons = 1;
}

message GetPersonsByIds {
  repeated string ids = 1;
}

message Empty {

}


service PersonService {
    rpc Create(Person) returns (Person);
    rpc GetByIds(GetPersonsByIds) returns (PersonList);
}